packages/block-editor/src/hooks/align.js

```diff
@@ -138,8 +138,8 @@ export const withToolbarControls = createHigherOrderComponent(
 
 		return [
 			validAlignments.length > 0 && props.isSelected && (
-				<BlockControls key="align-controls">
-					<BlockAlignmentToolbar
+				<BlockControls key="align-controls" group="block">
+					<BlockAlignmentControl
```

Comment:
`BlockAlignmentControl ` component needs some more debugging as it is responsible for the error reported in https://github.com/WordPress/gutenberg/pull/29247#issuecomment-795304103. If I comment out this component then everything else works properly.

Reply:
It looks like, `BlockAlignmentControl` doesn't cause the issue but something that is wrapping `BlockEdit`. When I replaced this component with a simple `span` I could observe exactly the same issue.

Reply:
All e2e tests pass so I doubt it's an issue but I haven't tried. It explodes in the mobile up just after you focus blocks like Pullquote, so it's easy to reproduce.